dev/TabView/TabView.xaml

```diff
@@ -104,7 +106,7 @@
         <Setter Property="ItemsPanel">
             <Setter.Value>
                 <ItemsPanelTemplate>
-                    <StackPanel Orientation="Horizontal" Padding="1,0,0,0" />
```

Comment:
Why was the padding moved down a level?

Reply:
Because ItemsStackPanel sadly doesn't have a padding property. :)

Reply:
That's weird.  But a good reason!